db/migrate/20201116134817_add_status_to_monitoree_counts.rb

```diff
@@ -0,0 +1,9 @@
+class AddStatusToMonitoreeCounts < ActiveRecord::Migration[6.0]
+  def up
+    add_column :monitoree_counts, :status, :string, default: 'Missing'
```

Comment:
in what situation would the status be missing?

Reply:
I can't think of one, but isn't it good practice to have a `default`?